src/xxh3.rs

```diff
@@ -20,6 +20,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
+#![allow(clippy::cast_possible_truncation)]
```

Comment:
Instead of allowing it for the whole file, can you move these to just the two functions that need it?

Reply:
Sure